### ISSUE TITLE
Properties on "module" objects get renamed

### ISSUE THREAD

**Issue by dariajung** [angular/ts-minify#30]
`import base = require('./base');`

-------
base.ts

```
export class Foo {
   . . .
}
```

`base` is some object that represents the `base` module. `base.Foo` is seen as a property access expression but we don't want to rename the `Foo` class name.

Possible routes:

- [ ] Forbid `import * from 'foo'` syntax
- [ ] Detect `import * from 'foo'` and understand `foo` is a "module" bundle object in subsequent uses of `foo`
- [ ] Rename all identifiers (like class names, etc.)
- [ ] See if TypeScript has some sort of type/symbol information for modules

**Comment by dariajung** [comment]
For a program `a.ts` and `b.ts`

`a.ts`

```
export class Foo {
  bar: string;
}
```

`b.ts`

```
import a = require('./a.ts');
new a.Foo();
```

We can get the symbol of the type information of an expression for a property access expression, which tells us `a`'s name is `name: '"../../test/input/a"'`. This tells us the `a` isn't a type, but a module.

**Comment by rkirov** [comment]
I think this one is resolved with #31

**Comment by dariajung** [comment]
Closing
